Bridges/HttpKernel.php

```diff
@@ -290,7 +289,8 @@ protected function mapResponse(SymfonyResponse $syResponse, $stdout='')
 
         // get contents
         ob_start();
-        if ($syResponse instanceof SymfonyStreamedResponse) {
+        $content = $syResponse->getContent();
+        if ($content === false) {
```

Comment:
Can we add a test case where the kernel returns a StreamedResponse to ensure the old functionality works as expected?

Reply:
Where does the false come from? See 
https://github.com/symfony/http-foundation/blob/master/StreamedResponse.php#L102

Reply:
Nvm, got it.

Reply:
Also how does it handle other types of responses without content? e.g. RedirectResponse? I think we definitely need test cases with some different types of responses to make sure this works as expected.

Reply:
Responses without content return null so that would be fine.
Actually, the `else` branch will work also for non-streamed responses, because that's what Symfony does in `index.php` (headers + body instead of just body though). Maybe we could just always call `sendContent()` and work with the output buffers? Then php-pm doesn't have to know about specific kinds of responses.
From the default `index.php`:
```php
$response = $kernel->handle($request);
$response->send();
```

Reply:
Good idea - I like it! There is no reason the HttpKernel needs to distinguish between response types, it just needs to buffer the content. Its also a step in the right direction if we want to implement real streamed responses later I think.